programs/gateway/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "gpl-gateway"
```

Comment:
nit: should we call it `gpl-civic-gateway` ?

Reply:
✅